opm/input/eclipse/Schedule/Well/WellKeywordHandlers.cpp

```diff
@@ -332,6 +333,25 @@ void handleWCONINJH(HandlerContext& handlerContext)
     }
 }
 
+bool belongsToAutoChokeGroup(const Well& well, const ScheduleState& state) {
+    const auto& network = state.network.get();
```

Comment:
It is good to return false here when the network is not active to avoid going through the code following.

Reply:
Good point, will update.